src/main/scala/org/globalforestwatch/summarystats/annualupdate_minimal/AnnualUpdateMinimalSummary.scala

```diff
@@ -41,7 +41,7 @@ object AnnualUpdateMinimalSummary {
                ): Unit = {
         // This is a pixel by pixel operation
         val loss: Integer = raster.tile.loss.getData(col, row)
-        val gain: Boolean = raster.tile.gain.getData(col, row)
+        val gain: String = raster.tile.gain.getData(col, row)
```

Comment:
For clarify, maybe rename `gain` to `gainPeriod` here in this function.  Feel free to ignore.

Reply:
I agree.